src/index.js

```diff
@@ -48,7 +48,7 @@ function Postis(options) {
     send: function (opts) {
       var method = opts.method;
 
-      if (ready || opts.method === readyMethod) {
+      if (ready || opts.method === readyMethod && (targetWindow && typeof targetWindow.postMessage !== "undefined")) {
```

Comment:
@kristerkari I think we might still want to buffer the methods?

I'm planning another PR to destroy the channel after it is no longer needed. We could retouch this statement then and discard the send/buffer if channel has been explicitly destroyed.